recipes/testfixtures/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "4.10.0" %}
+{%set md5 = "64af471a82ab788c1f18e6c2b8bf0479" %}
+
+package:
+    name: testfixtures
+    version: {{ version }}
+
+source:
+    fn: testfixtures-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/t/testfixtures/testfixtures-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+
+test:
+    imports:
+        - testfixtures
+
+about:
+    home: http://pythonhosted.org/testfixtures/
+    license: MIT
+    summary: 'A collection of helpers and mock objects for unit tests and doc tests.'
+    doc_url: http://pythonhosted.org/testfixtures/
```

Comment:
PyPI says this. Though they have [readthedocs]( http://testfixtures.readthedocs.io/en/latest/ ) too. Not sure what we should prefer.